scripts/test/test-utils.sh

```diff
@@ -76,16 +77,16 @@ run_tests() {
 
   # Need to decompress images built with MENDER_COMPRESS_DISK_IMAGE=gzip before
   # running tests.
-  if [ -f deploy/${device_type}-${artifact_name}-mender.img.gz ]; then
+  if [ -f deploy/*.gz ]; then
     # sudo is needed because the image is created using docker-mender-convert
     # which sets root permissions on the image
-    sudo gunzip --force deploy/${device_type}-${artifact_name}-mender.img.gz
+    sudo gunzip --force deploy/*.gz
   fi
 
   # MEN-3051: Rename the files back to .sdimg, as the sdimg extension has meaning
   # in the test-infrastructure.
   for file in ${MENDER_CONVERT_DIR}/deploy/*.img; do
```

Comment:
This can probably also be changed to `${MENDER_CONVERT_DIR}/deploy/${converted_image_name}.img`.